accession-commons-mongodb/src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/mongodb/repository/BasicMongoDbAccessionedCustomRepositoryImpl.java

```diff
@@ -62,6 +66,9 @@ public BasicMongoDbAccessionedCustomRepositoryImpl(Class<DOCUMENT> clazz, MongoT
                 String duplicatedId = parseIdDuplicateKey(error).orElseThrow(() -> e);
                 duplicatedHash.add(duplicatedId);
             });
+        } catch (RuntimeException e) {
+            logger.error("Unexpected runtime exception on bulk insert", e);
```

Comment:
"in MongoDB bulk insert" for consistency with the message in `checkHashUniqueness`